shared/fight.cpp

```diff
@@ -42,19 +42,19 @@ int fightable::damage_calc(const magic &chosen_magic, const fightable &defender,
     double inner_damage_multiplier = 1 + char_data.stats.inner_power / INNER_BASE;
     double inner_damage_reduction;
     switch (chosen_magic.inner_property) {
```

Comment:
Can you please create another variable as the enum type(initialize it with the int) and switch on enum?
Also I think we can use chars in the db for this(enum can be a char too) - maybe pinyin initial (or change water to liquid or wood to timber etc)